pkg/pillar/hypervisor/kvm.go

```diff
@@ -68,5 +413,31 @@ func (ctx kvmContext) GetHostCPUMem() (types.HostMemory, error) {
 }
 
 func (ctx kvmContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
-	return nil, nil
+	// for more precised measurements we should be using https://github.com/cha87de/kvmtop
+	res := map[string]types.DomainMetric{}
+
+	proc, err := procfs.NewFS("/proc")
+	if err != nil {
+		return res, logError("can't access /procfs %v", err)
+	}
+
+	for dom, pid := range ctx.domains {
+		p, err := proc.Proc(pid)
+		if err != nil {
+			return res, logError("can't access stats for PID %d (%v)", pid, err)
```

Comment:
Don't you want to also log the name of the domain which can't be reported?

Reply:
Fixed